extra/release_tool.py

```diff
@@ -2460,17 +2460,30 @@ def log_and_run_cmd(cmd, workdir):
         fd.write(changelog_out.decode())
 
 
-def do_generate_release_notes(state):
-    """Generate Release Notes for the released tags"""
+def do_generate_release_notes(
+    base_dir, version_of_integration, prev_of_integration=None
+):
+    split = version_of_integration.split("..")
+    if len(split) > 1:
+        if prev_of_integration is not None:
```

Comment:
Suggestion:
```
if prev_of_integration is not None
```
=
```
if prev_of_integration
```

Reply:
Strictly speaking it doesn't matter in this case, but I would like it to match the default argument, and the lower variant will match `""` and `0` as well.